recipes/jaro-winkler/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "jaro-winkler" %}
+{% set version = "2.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/py3/j/jaro-winkler/jaro_winkler-{{ version }}-py3-none-any.whl
```

Comment:
Any chance to get this one built from the source instead of repacking the wheel?

Reply:
They don't have an sdist. That's why I used the noarch wheel. I could try to use the github source, but I thought the wheel would be better since it's from PyPI. I don't have a preference. What do you think?

Reply:
I'm always inclined to build from source. Sure, when it is a pure wheel that is kind of moot. If you don't mind let's use the GH source instead. If that is too much hassle, leave it be and let me know when this is ready for merging.